src/argparse.c

```diff
@@ -213,6 +262,9 @@ void neu_cli_args_init(neu_cli_args_t *args, int argc, char *argv[])
         }
     }
 
+    // load env
+    load_env(args, &log_level, &config_dir, &plugin_dir);
```

Comment:
command line arguments should take precedence over environment variables ?

Reply:
Parameter priority rules refer to the rules of EMQX。
https://docs.emqx.com/zh/enterprise/v5.1/configuration/configuration.html#%E9%85%8D%E7%BD%AE%E8%A6%86%E7%9B%96%E8%A7%84%E5%88%99